src/main/php/PDepend/Source/Language/PHP/PHPParserVersion80.php

```diff
@@ -433,7 +433,8 @@ protected function isNextTokenObjectOperator()
 
     protected function consumeObjectOperatorToken()
     {
-        return $this->consumeToken($this->tokenizer->peek() === Tokens::T_NULLSAFE_OBJECT_OPERATOR
+        return $this->consumeToken(
+            $this->tokenizer->peek() === Tokens::T_NULLSAFE_OBJECT_OPERATOR
             ? Tokens::T_NULLSAFE_OBJECT_OPERATOR
             : Tokens::T_OBJECT_OPERATOR
```

Comment:
```suggestion
                ? Tokens::T_NULLSAFE_OBJECT_OPERATOR
                : Tokens::T_OBJECT_OPERATOR
```
If so, I would indent the ternary branch (to mimic the indent we would have with an `if-else` statement).

Reply:
Done